src/index.js

```diff
@@ -1,21 +1,27 @@
 import './style.css';
+import { fillList } from './modules/fill-render.js';
+import { addTaskEvent, taskDescription, TaskList } from './modules/add-task.js';
```

Comment:
- [ ] [OPTIONAL] As I am not sure if you had implemented the edit functionality or if it was missed by the previous reviewer, so I am adding it as an optional requirement but I highly suggest you implement it as it is one of the core requirement of the project. see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213901259-8572139c-137a-4b78-b31b-4ed9979581ef.png)

I suggest you implement the edit functionality just like you have added the add task and delete task functionality  
hint: you can use an edit icon or use the click event to make the content editable and then update the task description when it is edited.

Reply:
Hello, @PinkMoon25 Thank you for your Review.
If this is optional, this PR should not be marked as required changes, I should be able to submit it.

Reply:
I have requested changes because it is one of the core requirements and I have marked it as [OPTIONAL]  because you either forgot to implement this functionality or it was missed by the previous reviewer and so it is up to you if you want to integrate this functionality or not, after making your decision you can open a new pr and it should be approved.